composer.json

```diff
@@ -48,8 +50,12 @@
     },
     "scripts": {
         "ci:php:lint": "find *.php Classes/ Configuration/ Tests/ -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
+        "ci:php:sniff": "phpcs Classes Tests",
+        "ci:ts:lint": "typoscript-lint -c Configuration/TsLint.yml --ansi -n --fail-on-warnings -vvv Configuration/TypoScript/",
```

Comment:
We can drop the trailing slash here.

Reply:
Since it's a folder, I would prefer to keep that trailing slash.

Reply:
IIRC, some tools automatically add the slash, leading to double slashes in the output in cases of validation errors. (I don't remember which of the tools I use in CI were affected, though.)